### PR TITLE
PIM-6023: Add a behat step allowing us to hover an element

### PR DESCRIPTION
**Pull request by ClementGautier**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | :negative_squared_cross_mark:
| Added Behats                      | :white_check_mark:
| Changelog updated                 | :negative_squared_cross_mark:
| Review and 2 GTM                  | In progress
| Micro Demo to the PO (Story only) | :negative_squared_cross_mark:
| Migration script                  | :negative_squared_cross_mark:
| Tech Doc                          | :negative_squared_cross_mark:

### CODE REVIEW

File: features/Context/WebUser.php [hunk 1]
```diff
@@ -1443,6 +1443,23 @@ public function iPressTheButton($button)
     }
 
     /**
+     * @param string locator
+     *
+     * @When /^I hover over the element "([^"]*)"$/
```
Comment (grena) [inline features/Context/WebUser.php]: It's weird to insert some kind of CSS related in Gherkin. Can't we name it explicitly "I hover the icon" or I don't know what it is, then, **internally**, use this method?

Comment (anaelChardan) [inline features/Context/WebUser.php]: I agree with pipou

Comment (ClementGautier) [inline features/Context/WebUser.php]: I personally disagree. It is easier this way to maintain: Creating specific step is counter productive IMO and lead to what we have actually (meaning more gherkin than actual code to test).

Comment (anaelChardan) [inline features/Context/WebUser.php]: Ok :-), Good explanation :-), I GTM

File: features/Context/WebUser.php [hunk 2]
```diff
@@ -1443,6 +1443,23 @@ public function iPressTheButton($button)
...
+     */
+    public function iHoverOverTheElement($locator)
+    {
+        $session = $this->getSession();
+        $element = $session->getPage()->find('css', $locator);
+
+        if (null === $element) {
```
Comment (phaseinducer) [inline features/Context/WebUser.php]: Please spin instead of doing this.

**Pull request merged**